.circleci/config.yml

```
@@ -39,15 +39,32 @@ jobs:
             export PATH=$GOPATH/bin:$PATH
             echo "mode: set" > coverage.txt
             for pkg in $(go list ./... | grep -v vendor); do
-                go test -race -timeout 600s -coverprofile=$(echo $pkg | tr / -).coverage $pkg
+                go test -race -timeout 600s  --coverpkg="$(go list ./... | tr '\n' ',')" -coverprofile=$(echo $pkg | tr / -).coverage $pkg
             done
             grep -h -v "^mode:" *.coverage >> coverage.txt
             rm -f *.coverage
             bash <(curl -s https://codecov.io/bash)
 
+
+  test-go110:
+    docker:
+      - image: circleci/golang:1.10.3
+    environment:
+      GOPATH: /home/circleci/.go_workspace
+    working_directory: /home/circleci/.go_workspace/src/github.com/loadimpact/k6
+    steps:
+      - checkout
+      - run:
+          name: Run tests and code coverage
+          command: |
+            export PATH=$GOPATH/bin:$PATH
+            for pkg in $(go list ./... | grep -v vendor); do
```

```suggestion
            go test -race -timeout 600s ./...
```
If we don't calculate code coverage, can't we just run the whole tests suite at once, it should be faster than running individual packages...

---

I don't know if circleci is able to handle it or if it won't break in the future, but I guess it would be useful as a warning if it works currently

---

Yeah, we can always roll it back...